### PR TITLE
Add mysqlclusters/finalizers to RBAC

### LINKED ISSUE #203: openshift 3.10 cant not create mysql cluster
## Versions

**MySQL Operator Version**:
0.2
**Environment**:
- **Kubernetes version** (use `kubectl version`):
ocp 3.10(1.10)
- **Cloud provider or hardware configuration**:
openstack
- **OS** (e.g. from /etc/os-release):
rhel 7.5
- **Kernel** (e.g. `uname -a`):

- **Others**:

## What happened?
```
1 controller.go:289] error syncing 'mysql-cluster/mysql': creating root password Secret: secrets "mysql-root-password" is forbidden: cannot set blockOwnerDeletion if an ownerReference refers to a resource you can't set finalizers on: User "system:serviceaccount:mysql-operator:mysql-operator" cannot update mysqlclusters/finalizers.mysql.oracle.com in project "mysql-cluster", <nil>
```

### PR DESCRIPTION
**Pull request by prydie**
Fixes #203

### CODE REVIEW

File: mysql-operator/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mysql-operator
-version: 0.2.0
+version: 0.2.1
```
Comment (owainlewis) [inline mysql-operator/Chart.yaml]: This is a catch 22 change where we need a release created or master will be broken for a while.

Comment (prydie) [reply mysql-operator/Chart.yaml]: The chart version is independent of the release version.

Comment (owainlewis) [reply mysql-operator/Chart.yaml]: Just realised and was about to undo the comment. Thanks.

Comment (owainlewis) [reply mysql-operator/Chart.yaml]: My mistake. This is actually fine.

**Pull request merged**